version-case.scrbl

```diff
@@ -25,11 +25,11 @@ with both the old and new @racket[unit] libraries.
 
 @racketblock[
 (module some-sample-unit-code mzscheme
-  (require (planet "version-case.ss" ("dyoo" "version-case.plt" 1))
+  (require version-case
            (lib "mred.ss" "mred"))
   
-  ;; Small test code to see that we can write unit-dependent code
-  ;; that still runs under both 360 and 369.
```

Comment:
Dropping pre-racket support means that we can't do what this comment says.

Reply:
Do you have any other good example?

Reply:
I changed the example.